content/applications/finance/fiscal_localizations/kenya.rst

```diff
@@ -7,114 +7,320 @@ Kenya
+
+#. The user must match the fetched vendor bill with an existing product. Each vendor bill must
+   contain a product to be confirmed and sent to eTIMS later on.
+#. Odoo looks at the existing purchase order lines matching the product and partner (if any). The
+   user must select the correct related purchase order line from the
+   :guilabel:`Purchase Order Line`: dropdown list. The received quantities on the purchase order
+   must be on the same as those indicated on the vendor bill. If not, the vendor bill cannot be sent
+   to eTIMS (indicated by a yellow banner).
+
+   - If no existing purchase order lines match the fetched invoice lines, users can create a
+     purchase order based on the unmatched lines. The resulting stock move *must* be validated. Once
+     done, users can confirm and send the bill to eTIMS.
+
+#. Make sure to set a method in the :guilabel:`eTIMS Payment Method` field of the bill to send the
+   vendor bill to the KRA.
+#. Once all steps are completed, the vendor bill can be sent by clicking :guilabel:`Send to eTIMS`.
+   When the vendor bill has been confirmed on eTIMS, the **KRA invoice number** can be found under
+   the :guilabel:`eTIMS Details`.
+
+.. image:: kenya/purchase-order-lines.png
+   :alt: Bill registration steps.
+
+Invoicing
+=========
+
+.. note::
+   The KRA does *not* accept sales if the product is not in stock.
+
+This is the **advised sales flow** in Odoo when selling:
+
+#. Create a **sales order**;
+#. :guilabel:`Validate` the invoice;
+#. :guilabel:`Confirm` the invoice;
+#. Click :guilabel:`Send and print`, and then tick the :guilabel:`Send to eTIMS` box.
+#. Finally, click :guilabel:`Send & print` to send the invoice.
+
+Once the invoice has been sent and signed by the KRA, the following information can be found on
+it:
+
+- **KRA invoice number**;
+- Mandatory KRA invoice fields, such as **SCU information**, **date**, **SCU ID**, **receipt
+  number**, **item count**, **internal date**, and **receipt signature**;
+- The **KRA tax table**;
+- A unique **KRA QR code** for the signed invoice.
+
+Imports
+=======
+
+Customs imports codes are **automatically** fetched from the KRA eTIMS API servers **every day**.
+These codes can be **manually** fetched by enabling :doc:`../../general/developer_mode` and going to
+:menuselection:`Settings --> Technical --> Automation: Scheduled Actions --> KE eTIMS: Receive
+Customs Imports from the OSCU`. Here, click :guilabel:`Run Manually` to fetch the codes. Import
+items can be found under :menuselection:`Accounting --> Vendors --> Customs Imports`.
+
+The following steps are required to send and have **customs imports** signed by the KRA:
+
+#. The customs import is fetched automatically from the KRA.
+#. Users *must* match the imported item with an existing registered product (or create a product if
+   no related product exists) in the :guilabel:`Product` field.
+#. Set a vendor in the :guilabel:`Partner` field.
```

Comment:
I don't see this field in the Customs import list?

Reply:
If there are no imports in your runbot, you can follow the flow here: https://app.excalidraw.com/l/65VNwvy7c4X/1aCSVGqyEMV